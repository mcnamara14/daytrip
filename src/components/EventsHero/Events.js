import React from 'react';
import './EventsHero.css';

export const EventsHero = () =>  {
  return (
    <div className="eventsHeroContainer">     
      <article className="eventsHeroTextBox">
        <h1>Rockies vs Reds</h1>
        <h4><span>Coors Field</span></h4>
        <button>Select Event</button>
      </article>
    </div>
  );
};

export default EventsHero;
