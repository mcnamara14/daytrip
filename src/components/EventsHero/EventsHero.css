.eventsHeroContainer {
  height: 575px;
  z-index: -1;
  position: relative;
  background: url('./assets/events-hero-bg-v2.jpg');
  background-size: cover;
}

.eventsHeroTextBox {
  width: 415px;
  text-align: left;
  position: absolute;
  left: 8.5%;
  bottom: 100px;
}

.eventsHeroTextBox button {
  background: #d9492c;
  height: 45px;
  text-align: center;
  color: #fff;
  border: none;
  border-radius: 3px;
  padding: 0 60px;
  font-size: 1rem;
  margin-top: 30px;
}
