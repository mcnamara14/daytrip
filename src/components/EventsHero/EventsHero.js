import React from 'react';
import './EventsHero.css';

export const EventsHero = () =>  {
  return (
    <div className="eventsHeroContainer">     
      <article className="eventsHeroTextBox">
        <h1>We plan your day for you</h1>
        <h4>
          <span>
            Select an event. Choose a type of restaurant and bar. 
            We will do the rest.
          </span>
        </h4>
        {/* <button>Select Event</button> */}
      </article>
    </div>
  );
};

export default EventsHero;
