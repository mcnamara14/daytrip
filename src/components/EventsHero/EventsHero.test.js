import React from 'react';
import ReactDOM from 'react-dom';
import EventsHero from './EventsHero';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  
});
