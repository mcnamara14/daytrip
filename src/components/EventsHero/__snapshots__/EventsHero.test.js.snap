// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <article
        className="eventsHeroTextBox"
>
        <h1>
                We plan your day for you
        </h1>
        <h4>
                <span>
                        Select an event. Choose a type of restaurant and bar. We will do the rest.
                </span>
        </h4>
</article>,
      "className": "eventsHeroContainer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            We plan your day for you
</h1>,
          <h4>
            <span>
                        Select an event. Choose a type of restaurant and bar. We will do the rest.
            </span>
</h4>,
        ],
        "className": "eventsHeroTextBox",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "We plan your day for you",
          },
          "ref": null,
          "rendered": "We plan your day for you",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <span>
              Select an event. Choose a type of restaurant and bar. We will do the rest.
</span>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Select an event. Choose a type of restaurant and bar. We will do the rest.",
            },
            "ref": null,
            "rendered": "Select an event. Choose a type of restaurant and bar. We will do the rest.",
            "type": "span",
          },
          "type": "h4",
        },
      ],
      "type": "article",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <article
          className="eventsHeroTextBox"
>
          <h1>
                    We plan your day for you
          </h1>
          <h4>
                    <span>
                              Select an event. Choose a type of restaurant and bar. We will do the rest.
                    </span>
          </h4>
</article>,
        "className": "eventsHeroContainer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              We plan your day for you
</h1>,
            <h4>
              <span>
                            Select an event. Choose a type of restaurant and bar. We will do the rest.
              </span>
</h4>,
          ],
          "className": "eventsHeroTextBox",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "We plan your day for you",
            },
            "ref": null,
            "rendered": "We plan your day for you",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span>
                Select an event. Choose a type of restaurant and bar. We will do the rest.
</span>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Select an event. Choose a type of restaurant and bar. We will do the rest.",
              },
              "ref": null,
              "rendered": "Select an event. Choose a type of restaurant and bar. We will do the rest.",
              "type": "span",
            },
            "type": "h4",
          },
        ],
        "type": "article",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
